exercises/concept/boutique-inventory/.docs/hints.md

```diff
@@ -13,14 +13,18 @@
 
 - There is a [built-in function][enum-filter] for filtering enumerables.
 
-## 3. Increment the item's quantity
+## 3. Update item names
+
+- There is a [built-in function][enum-map] for replacing every element in an enumerable with another element.
```

Comment:
I wouldn't say "replacing" for `Enum.map` because it feels like the original element is discarded regardless of its value, maybe "transforming" instead?